plasmapy/particles/tests/test_decorators.py

```diff
@@ -344,10 +343,6 @@ def test_preserving_signature_with_stacked_decorators(decorator1, decorator2):
     assert undecorated_signature == decorated_signature_1_2 == decorated_signature_2_1
 
 
-@pytest.mark.xfail(
-    condition=sys.version_info < (3, 9),
```

Comment:
I assume this passes now?

Reply:
This was a test failure specific to Python 3.8, and we now require Python ≥3.9, so we're all set with this.